docs/src/content/guides/events.md

```diff
@@ -301,3 +284,51 @@ For simple events, it may be desireable to bypass Victory's event system. To do
     }
   />
 ```
+
+## Events on Custom Components
+
+For custom SVG components, Victory's event system is not the only option. It's possible to bypass it altogether in favor of using native React events on the custom SVG components. This might be preferable when lots of custom styling and interactions are needed.
+
+Notice how the `circle` component in this example is a basic SVG element with React event props rather than a Victory component.
+
+```playground_norender
+const ScatterPoint = ({ x, y, datum }) => {
+  const [selected, setSelected] = React.useState(false);
+  const [hovered, setHovered] = React.useState(false);
```

Comment:
I noticed that the other examples are using class components with `setState` - does it matter if my examples use newer syntax?

Reply:
Please do! The old syntax is just the library and docs showing their age.